schema/pgsql/upgrades/1.1.2.sql

```diff
@@ -137,3 +137,8 @@ BEGIN
   RETURN (100 * (total_time - problem_time)::decimal / total_time)::decimal(7, 4);
 END;
 $$;
+
+CREATE INDEX idx_history_event_time_event_type ON history(event_time, event_type);
```

Comment:
To discuss: do we want to use [`CREATE INDEX CONCURRENTLY`](https://www.postgresql.org/docs/current/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY) here instead? It will scan the table twice but allows for concurrent modifications of the table (reading is already possibly without `CONCURRENTLY`).